### DISCUSSION TITLE
QZ to Kinomap

### DISCUSSION THREAD

**Discussion by luvictu** [cagnulein/qdomyos-zwift#2455] [Q&A] [answered]
hi Roberto,

I haven't found any information about QZ being able to connect with Kinomap just in the same way it does with Zwift? Is this something currently available or maybe planned for the future?
Thank you!!

**Comment by cagnulein** [comment]
it's already available. just use 1 device for qz and 1 for kinomap. be sure
to select ftms on kinomap
let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 17 lug 2024 alle 13:01 luvictu ***@***.***> ha
scritto:

**Reply by luvictu** [reply]
Awesome, will try this and let you know, thanks a bunch!

**Comment by luvictu** [comment, marked as answer]
hey Roberto, I tested it and all works like a charm. Too bad that my Bowflex treadmill would not accept auto-incline but otherwise really cool to be able to connect it and send the speed info to Kinomap.
Cheers!

**Reply by cagnulein** [reply]
@luvictu that's great! remember to rate the app on the store and if you can also joining the swag bag in the app, I need the support from everyone to keep this app alive! Thanks

**Reply by TWilky** [reply]
Hi, I've a joroto x4s, how do I connect it to kinomaps? I've one device that links qz to the joroto bike, then a separate device for kinomaps. When I go to kinomaps I go to add equipment, then bikes, there is qz listed as a brand, but when I click it I don't see the joroto bike (seems to be a list of other preset bikes). Lol clearly I'm just doing the wrong thing. Could anyone point me in the right direction please?

**Comment by cagnulein** [comment]
You have to select ftms bike in kinomap and then the qz phone

Let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 20 ott 2025 alle 21:23 TWilky ***@***.***> ha
scritto:

**Comment by TWilky** [comment]
Hey Roberto,

Can't see an option to add a ftms bike on kinomaps, when I go to bikes,
there is either add an optical sensor or add the machine (brand of bikes).
Is there somewhere else I've to go. Sorry

CheerS

**Comment by cagnulein** [comment]
In the brands there is the ftms one

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 20 ott 2025 alle 21:49 TWilky ***@***.***> ha
scritto:

> Hey Roberto,
>
> Can't see an option to add a ftms bike on kinomaps, when I go to bikes,
> there is either add an optical sensor or add the machine (brand of bikes).
> Is there somewhere else I've to go. Sorry
>
> CheerS
>

**Comment by TWilky** [comment]
Ahh, so it is, sorry. I've selected it, it then connects to my other device
but says main data not received. I presume I'm doing something wrong on the
qz app. In the qz app, under bike options I've selected the joroto for in
the ftms.
Thanks

**Comment by cagnulein** [comment]
The ftms bike setting under the bike options in qz must be disabled

Do you see correctly the metrics on qz from your bike?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 20 ott 2025 alle 21:57 TWilky ***@***.***> ha
scritto:

> Ahh, so it is, sorry. I've selected it, it then connects to my other
> device
> but says main data not received. I presume I'm doing something wrong on
> the
> qz app. In the qz app, under bike options I've selected the joroto for in
> the ftms.
> Thanks
>

**Comment by TWilky** [comment]
Yeah, they are coming up on my phone but not on kinomaps. You can see in
the attached screenshot.

**Comment by cagnulein** [comment]
I can't see the screenshot, you have to upload on github web interface
directly
Thanks

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 20 ott 2025 alle 22:05 TWilky ***@***.***> ha
scritto:

> Yeah, they are coming up on my phone but not on kinomaps. You can see in
> the attached screenshot.
>

**Reply by TWilky** [reply]
Ah, the screenshot was just the home screen of the qz app when spinning on the bike, the speed cadence etc was going up but nothing on the kinomaps. Is there something I need to select in the options.

**Comment by TWilky** [comment]
<img width="1080" height="2400" alt="Screenshot_20251020-210324" src="https://github.com/user-attachments/assets/23aab3b2-6642-4f57-bdf8-e5891587d416" />
<img width="1080" height="2400" alt="Screenshot_20251020-210306" src="https://github.com/user-attachments/assets/5a06deba-2cf4-44e1-ab4d-038d27445b29" />

**Reply by cagnulein** [reply]
Disable the ftms bike setting there. It must be set to disabled

**Reply by TWilky** [reply]
Fantastic. I got there. Thank you for your patience!!
